src/windowsPtyAgent.ts

```diff
@@ -162,16 +162,7 @@ export class WindowsPtyAgent {
     this._outSocket.readable = false;
     // Tell the agent to kill the pty, this releases handles to the process
     if (this._useConpty) {
-      this._getConsoleProcessList().then(consoleProcessList => {
-        consoleProcessList.forEach((pid: number) => {
-          try {
-            process.kill(pid);
-          } catch (e) {
-            // Ignore if process cannot be found (kill ESRCH error)
-          }
-        });
-        (this._ptyNative as IConptyNative).kill(this._pty, this._useConptyDll);
-      });
+      (this._ptyNative as IConptyNative).kill(this._pty, this._useConptyDll);
```

Comment:
I think this means we can remove `conpty_console_list_agent.ts` and `win/conpty_console_list.cc`

Reply:
Yeah left it as such incase you had use cases in future. Happy to remove them.

Reply:
Let's remove if we don't need

Reply:
I retain the old tree kill logic for `useConpty` mode while `useConptyDll` mode uses the new logic. This allows for a possible fallback incase things go horribly wrong. We can merge the two paths once we gain confidence.